invoice/src/main/java/org/killbill/billing/invoice/optimizer/InvoiceOptimizerExp.java

```diff
@@ -80,7 +80,7 @@ public AccountInvoices getInvoices(final InternalCallContext callContext) {
         //
         final LocalDate beCutoffDt = isMaxInvoiceLimitSet ? cutoffDt.minus(maxInvoiceLimit) : null;
         final List<Invoice> existingInvoices = new LinkedList<Invoice>();
-        final List<InvoiceModelDao> invoicesByAccount = invoiceDao.getInvoicesByAccount(false, cutoffDt, null, true, callContext);
+        final List<InvoiceModelDao> invoicesByAccount = invoiceDao.getInvoicesByAccount(false, cutoffDt, null, true, false, callContext); //TODO_1952 - non API call, so passing includeTrackingIds=false. Revisit
```

Comment:
I think this is true as well - comment can go. 

The only time we really need those in the code is when we generate an invoice, and those tracking IDs are explicitly fetched as shown [here](https://github.com/killbill/killbill/blob/master/invoice/src/main/java/org/killbill/billing/invoice/usage/RawUsageOptimizer.java#L92). So any internal use of invoices inside KB should not need those.